.github/workflows/tf-checks.yml

```diff
@@ -0,0 +1,12 @@
+name: tf-checks
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+  workflow_dispatch:
+jobs:
+  basic:
+    uses: clouddrove/github-shared-workflows/.github/workflows/tf-checks.yml@1.2.4
+    with:
+      working_directory: './examples/'
```

Comment:
Add working directory proper for basic and complete both

Reply:
done sir